src/static/css/module/loan-comparison.less

```diff
@@ -364,63 +364,138 @@
   --------------------------- */
 
 .comparison-results {
-  margin-top: 60px;
+  .expandable {
+    .respond-to-min(@tablet-min, {
+      overflow: hidden;
+    });
+  }
+  .expandable_header {
+    margin-bottom: @grid_gutter-width/2;
+    background: @gray-10;
+    .respond-to-min(1000px, {
+      margin-bottom: @grid_gutter-width;
+    });
+  }
+  .expandable_link {
+    position: relative;
+    z-index: 1;
+  }
+  .expandable__padded,
+  .expandable__padded.expandable__expanded {
+    background: @white;
+  }
+  .expandable__padded .expandable_content {
+    margin-right: 0;
+    margin-left: 0;
+    background: @white;
+    &:before,
+    &:hover {
```

Comment:
There was some previous discussion in flapjack about providing options for background colors on `.expandable-groups` -- we might want to suggest that for `.expandable__padded` as well. Another approach could be moving this into a modifier that inherits background color. But I think it's fine to do the override here unless/until we need it somewhere else.